docs/reference/platforms/android/gradle.rst

```diff
@@ -102,6 +102,22 @@ You can specify a background color for the splash screen using the
 
 Android projects do not support installer images.
 
+Colors
+======
+
+In addition to ``splash_background_color``, Android projects allow for
+configuration of an ``accent_color``, ``primary_color``, and
+``primary_color_dark``.
```

Comment:
There's no need to list all the things that can be configured just before you list all the things that can be configured :-)
```suggestion
Android allows for some customization of the colors used by your app:
```